submissions/ik-web/html-css-popup/css/app.css

```diff
@@ -0,0 +1,256 @@
+/* Common styles
+------------------------- */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    outline: none;
+    font-family: sans-serif;
+}
+
+a {
+    text-decoration: none;
+}
+
+ul li {
+    position: relative;
+    list-style: none;
+}
+
+img {
+    display: block;
+    width: 100%;
+}
+
+/* Variables
+------------------------- */
+:root {
+    --text-color: #616161;
+    --text-hover-color: #ffffff;
+    --icon-bg-color: #cfcccc;
+    --icon-border-color:  #858383;
+    --btn-border-color: #ffffff;
+    --btn-hover-color: #858383;
+    --menu-bg-color:#cfcccc;
+}
+
+/* Body container
+------------------------- */
+body {
+    width: 100vw;
+    height: 100vh;
+
+    font-size: 18px;
+    background: #fff url('../img/main-bg.jpg') center center/cover no-repeat;
+}
+
+/* Nav container
+------------------------- */
+.nav {
+    padding: 10px 50px 10px 0;
+}
+
+.nav-inner {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.nav-item {
+    display: block;
+    margin: 0 10px;
+    padding: 5px;
+    color: var(--text-color);
+}
+
+/* Nav links
+------------------------- */
+.nav-link:after {
+    content:'';
+    display: block;
+    position: absolute;
+    bottom: 3px;
+    left: 50%;
+    transform: translateX(-50%);
+    width: 0;
+    height: 2px;
+    background-color: var(--text-color);
+
+    transition: width .15s linear;
+}
+
+.nav-link:focus:after,
+.nav-link:hover:after {
+    width: 70%;
+}
+
+/* Nav icons
+------------------------- */
+.nav-icon img {
+    width: 30px;
+    height: 30px;
+    border-radius: 100%;
+}
+
+label {
+    cursor: pointer;
+}
+
+.nav-icon .nav-user {
+    width: 40px;
+    height: 40px;
+} 
+
+.nav-icon {
+    border-radius: 100%;
+    transition: background-color .15s linear;
+}
+
+.nav-icon:focus,
+.nav-icon:hover {
+    background-color:var(--icon-bg-color);
+}
+
+/* Popup menu container
+------------------------- */
+#popup-btn_more,
```

Comment:
would be better to use classes for styles instead of ID selectors. Could you please fix it for all selectors? ID should be used only for JS

Reply:
I want to clarify the situation. This ID was created to bind the label to the checkbox. Do I need to add a class for the checkbox to work with it in CSS or can I use the assigned ID?

Reply:
I fixed those selectors and thanks for all the information.